sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonPlugin.java

```diff
@@ -105,6 +107,12 @@ public void define(Context context) {
       PythonRuleRepository.class);
 
     if (context.getRuntime().getProduct() != SonarProduct.SONARLINT) {
+      if (context.getRuntime().getApiVersion().isGreaterThanOrEqual(SQ_MIN_VERSION_ANALYSIS_WARNINGS)) {
```

Comment:
I would extract `context.getRuntime()` into a dedicated variable to not call it twice in a row.